apps/communications/contacts/style/app.css

```diff
@@ -552,6 +552,16 @@ nav[data-type="scrollbar"] ol > li > * {
   display: block !important;
 }
 
+.selecting.disable-fb li[data-fb-uid] {
```

Comment:
bettter to name this class disable-fb-items and put a comment indicating its purpose